pyproject.toml

```diff
@@ -13,7 +13,6 @@ requires-python = ">=3.8"
 license = {text = "Apache 2.0"}
 dependencies = [
     "h5py>=3.7.0",
-    "mpi4py>=3.1.4",
```

Comment:
I think we shouldn't silently remove these. At minimum we should add a proper import error message.